edgexfoundry/edgex-compose
# feat: Add MessageBus config for all core/support services

This is in preparation from when these service's publish Service Metrics or System Events to the MessageBus

closes #258

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>


## Testing Instructions
Run `make gen no-secty`
Verify all core/support services have the `MESSAGEQUEUE_HOST=edgex-redis` override
Run `make gen no-secty mqtt-bus`
Verify all core/support services have the following env overrides:
- MESSAGEQUEUE_AUTHMODE: none
- MESSAGEQUEUE_HOST: edgex-mqtt-broker
- MESSAGEQUEUE_OPTIONAL_CLIENTID: <service-name>
- MESSAGEQUEUE_PORT: '1883'
- MESSAGEQUEUE_PROTOCOL: tcp
- MESSAGEQUEUE_TYPE: mqtt
